src/main/java/org/jfrog/bamboo/task/ArtifactoryGenericDeployTask.java

```diff
@@ -125,15 +129,37 @@ public TaskResult execute(@NotNull TaskContext taskContext) throws TaskException
         return TaskResultBuilder.newBuilder(taskContext).success().build();
     }
 
+    @Override
+    protected ServerConfigBase getUsageServerConfig() {
+        return buildInfoHelper.getServerConfig();
+    }
+
+    @Override
+    protected String getTaskUsageName() {
+        return "rt_upload";
```

Comment:
I suggest using a naming conversion that will make it easier to figure out which tasks are used.
For example, for the *ArtifactoryGenericDeployTask* we can use *generic_deploy*.